pkg/api/vlabs/validate_test.go

```diff
@@ -562,88 +564,33 @@ func Test_Properties_ValidateContainerRuntime(t *testing.T) {
 }
 
 func TestWindowsVersions(t *testing.T) {
-	p := getK8sDefaultProperties(true)
-	p.OrchestratorProfile.OrchestratorRelease = "1.6"
-	if err := p.Validate(false); err == nil {
-		t.Errorf(
-			"should error on invalid Windows version",
-		)
-	}
-
-	p = getK8sDefaultProperties(true)
-	p.OrchestratorProfile.OrchestratorRelease = "1.7"
-	if err := p.Validate(false); err != nil {
-		t.Errorf(
-			"should not error on valid Windows version: %v", err,
-		)
-	}
-
-	p = getK8sDefaultProperties(true)
-	p.OrchestratorProfile.OrchestratorVersion = "1.7.15"
-	if err := p.Validate(false); err != nil {
-		t.Errorf(
-			"should not error on valid Windows version: %v", err,
-		)
-	}
-
-	p = getK8sDefaultProperties(true)
-	p.OrchestratorProfile.OrchestratorVersion = "1.7.0"
-	if err := p.Validate(false); err == nil {
-		t.Errorf(
-			"should error on invalid Windows version",
-		)
-	}
-
-	p = getK8sDefaultProperties(true)
-	p.OrchestratorProfile.OrchestratorRelease = "1.8"
-	if err := p.Validate(false); err != nil {
-		t.Errorf(
-			"should not error on valid Windows version: %v", err,
-		)
-	}
-
-	p = getK8sDefaultProperties(true)
-	p.OrchestratorProfile.OrchestratorVersion = "1.8.10"
-	if err := p.Validate(false); err != nil {
-		t.Errorf(
-			"should not error on valid Windows version: %v", err,
-		)
-	}
-
-	p = getK8sDefaultProperties(true)
-	p.OrchestratorProfile.OrchestratorVersion = "1.8.11"
-	if err := p.Validate(false); err == nil {
-		t.Errorf(
-			"should error on invalid Windows version",
-		)
-	}
-
-	p = getK8sDefaultProperties(true)
-	p.OrchestratorProfile.OrchestratorRelease = "1.9"
-	if err := p.Validate(false); err != nil {
-		t.Errorf(
-			"should not error on valid Windows version: %v", err,
-		)
-	}
-
-	p = getK8sDefaultProperties(true)
-	p.OrchestratorProfile.OrchestratorVersion = "1.9.6"
-	if err := p.Validate(false); err != nil {
-		t.Errorf(
-			"should not error on valid Windows version: %v", err,
-		)
+	for _, version := range common.GetAllSupportedKubernetesVersionsWindows() {
```

Comment:
using a bunch of static versions was not maintainable, actually enumerating through known supported list now to test validation